lib/containers/utils.pm

```diff
@@ -41,20 +41,56 @@ sub test_seccomp {
     }
 }
 
-sub basic_container_tests {
-    my %args    = @_;
-    my $runtime = $args{runtime};
-    die "You must define the runtime!" unless $runtime;
+sub check_docker_firewall {
+    my $container_name = 'sut_container';
+    validate_script_output('ip a s docker0', sub { /state DOWN/ });
+    assert_script_run "firewall-cmd --list-all --zone=docker";
```

Comment:
We don't use `firewalld` on *<SLE15* but `SUSEfirewall2`.

Reply:
nice catch. check the update